docs/cog_guides/core.rst

```diff
@@ -0,0 +1,3393 @@
+
+Removes a mod role for this guild.
+
+Examples:
+    - ``[p]set removemodrole @Mods``
+    - ``[p]set removemodrole Loyal Helpers``
+
+**Arguments:**
+
+- ``<role>`` The role to remove from being a moderator.
+
+.. _core-command-set-serverfuzzy:
+
+"""""""""""""""
+set serverfuzzy
+"""""""""""""""
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set serverfuzzy 
+
+**Description**
+
+Toggle whether to enable fuzzy command search for the server.
+
+This allows the bot to identify potential misspelled commands and offer corrections.
+
+.. Note:: This can be processor intensive and may be unsuitable for larger servers.
+
+
+Default is for fuzzy command search to be disabled.
+
+.. _core-command-set-serverprefix:
+
+""""""""""""""""
+set serverprefix
+""""""""""""""""
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set serverprefix [prefixes...]
+
+.. tip:: Alias: ``set serverprefixes``
+
+**Description**
+
+Sets Red's server prefix(es).
+
+.. Warning:: This will override global prefixes, the bot will not respond to any global prefixes in this server.
+
+    This is not additive. It will replace all current server prefixes.
+
+Examples:
+    - ``[p]set serverprefix !``
+    - ``[p]set serverprefix "@Red "`` - Quotes are needed to use spaces. This uses a mention as the prefix.
+    - ``[p]set serverprefix ! ? .`` - Sets multiple prefixes.
+
+**Arguments:**
+
+- ``[prefixes...]`` The prefixes the bot will respond to on this server. Leave blank to clear server prefixes
+
+.. _core-command-set-showsettings:
+
+""""""""""""""""
+set showsettings
+""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set showsettings 
+
+**Description**
+
+Show the current settings for Red.
+
+.. _core-command-set-status:
+
+""""""""""
+set status
+""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set status <status>
+
+**Description**
+
+Sets Red's status.
+
+Available statuses:
+    - ``online``
+    - ``idle``
+    - ``dnd``
+    - ``invisible``
+
+Examples:
+    - ``[p]set status online`` - Clears the activity status.
+    - ``[p]set status invisible``
+
+**Arguments:**
+
+- ``<status>`` One of the available statuses.
+
+.. _core-command-set-streaming:
+
+"""""""""""""
+set streaming
+"""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]set streaming [(<streamer> <stream_title>)]
+
+.. tip:: Aliases: ``set stream``, ``set twitch``
+
+**Description**
+
+Sets Red's streaming status to a twitch stream.
+
+This will appear as ``Streaming <stream_title>`` or ``LIVE ON TWITCH`` depending on the context.
+It will also include a ``Watch`` button with a twitch.tv url for the provided streamer.
+
+Maximum length for a stream title is 128 characters.
+
+Leaving both streamer and stream_title empty will clear it.
+
+Examples:
+    - ``[p]set competing`` - Clears the activity status.
+    - ``[p]set competing London 2012 Olympic Games``
```

Comment:
Incorrect

Reply:
Added correct examples here.
Also adjusted the arguments to be in `<>` to indicate required, since that's how each individual argument is presented in the usage.